spyder/app/mainwindow.py

```diff
@@ -261,6 +266,7 @@ def __init__(self, options=None):
 
         qapp = QApplication.instance()
         if PYQT5:
+            # Enabling scpaling for high dpi
             qapp.setAttribute(Qt.AA_UseHighDpiPixmaps)

```

Comment:
I think the critical one is `HighDpiScaling`. No one has reported problems about this one :-)